### PR TITLE
Update the map legend

### PR DESCRIPTION
**Pull request by Gustry**
* The legend : 
screen shot 2017-01-23 at 15 20 41

* The report :
screen shot 2017-01-23 at 15 21 02

@lucernae : 
* We need to check why we don't get the same rounding. Numbers should match between the legend and the report.
* You need to add a new row I think for not exposed according to the new diagram : 
screen shot 2017-01-23 at 15 22 49

### CODE REVIEW

**Comment by lucernae** [comment]
@Gustry 
I remembered in the previous report, exposure type other than population, will not have rounding.
Like building count or roads length. We need to make sure that @timlinux and @Charlotte-Morgan are aware of this.

**Comment by Gustry** [comment]
Yes, you are right.

I was just reading the markdown document about rounding and I did not see any conditions.
https://github.com/inasafe/inasafe/blob/develop/docs/reporting-standards.md#legend-label-rounding

@Charlotte-Morgan Can ew have your input ? Thanks.

**Comment by Gustry** [comment]
Ok, I have disabled the rounding until we have some inputs.
I would like to merge this PR and work with @lucernae to include not exposed in the report.

File: safe/utilities/rounding.py [hunk 1]
```diff
@@ -6,6 +6,36 @@
 __revision__ = '$Format:%H$'
 
 
+def round_affected_number(
```
Comment (lucernae) [inline safe/utilities/rounding.py]: Thank's for moving this ;)

**Comment by lucernae** [comment]
LGTM.

I will use units semantic too in the reporting and postprocessors.

**Pull request merged**